library/src/test/java/com/datatorrent/lib/helper/OperatorContextTestHelper.java

```diff
@@ -44,22 +42,26 @@ protected DateFormat initialValue()
 
   };
 
-  public static class TestIdOperatorContext extends TestContext implements OperatorContext
+  public abstract static class TestIdOperatorContext extends TestContext implements OperatorContext
```

Comment:
The name of the class is not that important - IMO, it should not be visible at all.

Please explain what is the reason to have it exposed. I agree that in the future it may be necessary to provide builder style setters, but they can and should be provided on the builder class (can be the parent or a helper class). The builder class by itself should not implement OperatorContext interface. It should be mocked. I don't see why test cases should use MockOperatorContext instead of OperatorContext.